packages/api-headless-cms-ddb/src/operations/entry/index.ts

```diff
@@ -816,14 +911,12 @@ export const createEntriesStorageOperations = (
             ids: [params.id]
         });
 
-        return items
-            .filter(item => item.deleted !== true)
-            .map(item => {
-                return convertFromStorageEntry({
-                    storageEntry: item,
-                    model
-                });
+        return items.map(item => {
+            return convertFromStorageEntry({
```

Comment:
I see we've removed the `filter` call here. What was the reason? Just curious.

Reply:
This is a leftover from a previous PR: at the moment, we are removing deleted entries decorating the default use case.

https://github.com/webiny/webiny-js/blob/next/packages/api-headless-cms/src/crud/contentEntry/useCases/GetRevisionsByEntryId/GetRevisionsByEntryIdNotDeleted.ts